run_scripts/2_fetch_raw_data.py

```diff
@@ -70,43 +73,51 @@
     storage_client = storage.Client.from_service_account_json(google_cloud_credentials_file_path)
     credentials_bucket = storage_client.bucket(bucket_name)
     credentials_blob = credentials_bucket.blob(blob_name)
-    rapid_pro_token = credentials_blob.download_as_string().strip()
+    rapid_pro_token = credentials_blob.download_as_string().strip().decode("utf-8")
```

Comment:
Is this the right way round? i would have thought we needed decode and then strip?

Reply:
Both ways work, because strip works on bytes and strings, but rapid_pro only works on strings.